analytics/app/data/load.py

```diff
@@ -0,0 +1,32 @@
+import os
+from pathlib import Path
+
+from modyn.supervisor.internal.pipeline_executor.models import PipelineLogs
+
+EVAL_DIR = Path(os.environ["MODYN_EVAL_DIR"])
```

Comment:
Is there any way to make this part of the UI or at least provide it as a command line arg? I switched around between different directories quite a lot (not sure if everything breaks down when the pipelines stem from different datasets, I kept that separate), and always re-exporting an environment variable was a bit annoying

Reply:
CLI is possible, I'll add that. Adding it to the UI doesn't work as we need the data on the initial render for the ui callbacks to work... Plotly dash oddities I guess...